nservicebus/upgrades/5to6.md

```diff
@@ -654,13 +654,20 @@ The `NServiceBus.Callbacks` package has to be referenced only by the requesting
 
 snippet: 5to6-Callbacks
 
-In Version 6 the callback handling on [broker transports](/nservicebus/transports/#types-of-transports-broker-transports) is no longer based on the machine host name. In order to continue using callbacks when using a broker transport, a unique endpoint instance ID has to be provided:
+In Version 6 the callback handling on [broker transports](/nservicebus/transports/#types-of-transports-broker-transports) is no longer based on the machine host name. In order to continue using callbacks when using a broker transport, the endpoint needs to be uniquely addressable:
 
 snippet: 5to6-Callbacks-InstanceId
 
 NOTE: This ID should never be hardcoded, e.g. it can be read from the configuration file or from the environment (e.g. role ID in Azure), so that it can be changed without code changes and redeployment.
 
 
+## Uniquely addressable endpoint instances
+
+For certain scale out scenarios and callbacks, endpoint instances need to be uniquely addressable. Using `busConfiguration.ScaleOut().UniqueQueuePerEndpointInstance("-MyCustomSuffix");` would change the input queue name to include the specified suffix. With Version 6, the original input queue remains and an additional queue with the configured suffix is created. Uniquely addressable endpoints can consume messages from both (shared and unique) queues. Replies will automatically be sent to the instance specific queue. Configure an endpoint use an uniquely addressable queue with:
+
```

Comment:
I think most of this paragraph should be moved to http://docs.particular.net/nservicebus/messaging/handling-responses-on-the-client-side?version=callbacks_1#message-routing and we should link to that page. From the upgrade guide perspective I think it is enough to say "Please use this API".